web/src/composebox_typeahead.ts

```diff
@@ -1179,6 +1191,56 @@ export function content_typeahead_selected(
     return beginning + rest;
 }
 
+function will_produce_broken_stream_topic_link(word: string): boolean {
+    return invalid_stream_topic_regex.test(word);
+}
+
+function get_stream_name(syntax: string): string {
+    const start = syntax.lastIndexOf("#**");
+    const end = syntax.lastIndexOf(">");
+    return syntax.slice(start + 3, end);
+}
+export function replace_markdown_characters(text: string): string {
```

Comment:
You should have a blank line between functions.

Also, usually it's better to add new functions above the one function that uses them.

Reply:
Sure, done